### DISCUSSION TITLE
Update info not translated

### DISCUSSION THREAD

**Discussion by Hi-there-how-are-u** [sircharlo/meeting-media-manager#414] [Q&A] [answered]
I noticed that there was an update.
When installed the update I got english notification that it was done even if I have this translated in English.
What I can see this line is translated in Gitlocalize: An update to M³ was installed.
and also Find out more is translated but still it was in English?
![Meeting_Media_Manager_FsMwrU7l9A](https://user-images.githubusercontent.com/36486218/179392882-24d873b5-6dea-42d2-92ec-4f99f4565fa6.png)

**Comment by mtdvlpr** [comment]
Which language do you have selected in settings > application setup?

**Reply by Hi-there-how-are-u** [reply]
I have selected Swedish as language.

**Reply by mtdvlpr** [reply]
I can't seem to replicate the issue. Do you have some more examples of english phrases you see that should be translated? Can you show me your console logs (ctrl + shift + i -> console)?

**Reply by Hi-there-how-are-u** [reply]
That was the only thing i noticed. I can check the logs. Unfortunately I am not at home so it will take a week or two... Otherwise i can just wait for the next update to see if it occurs again.

**Reply by sircharlo** [reply]
Do you have more than one congregation configured in M³?

**Reply by Hi-there-how-are-u** [reply]
No I have only one congregation.

**Reply by Hi-there-how-are-u** [reply]
I tried now at another computer. I installed 22.5.7 version and I changed the language to Swedish and directly it starts download new version on the background.  After restart the program is in Swedish but still the message is in English.
But, can it be that the words are translated in the 22.7.1? And that´s why I get it in English? In that case it should be in English next time?

This was in the console:
DevTools failed to load source map: Could not parse content for file:///C:/Users/User/AppData/Local/Programs/Meeting%20Media%20Manager/resources/app.asar/popper.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///C:/Users/User/AppData/Local/Programs/Meeting%20Media%20Manager/resources/app.asar/bootstrap.js.map: Unexpected end of JSON input
DevTools failed to load source map: Could not parse content for file:///C:/Users/User/AppData/Local/Programs/Meeting%20Media%20Manager/resources/app.asar/h264-mp4-encoder.node.js.map: Unexpected end of JSON input

**Comment by sircharlo** [comment, marked as answer]
The issue is [here](https://github.com/sircharlo/meeting-media-manager/blob/1a5ca708edc553ddd2b452fa560bb333740bfce4/media-updater.js#L104): the post-update message is displayed before the user's settings are retrieved [here](https://github.com/sircharlo/meeting-media-manager/blob/1a5ca708edc553ddd2b452fa560bb333740bfce4/media-updater.js#L106). Therefore, the default or fallback language is displayed for that notification, that is, English. 

I'd have to move the post update notification to right after a congregation has been selected.
